lib/Debugger.Protocol/js_protocol.json

```diff
@@ -513,7 +513,7 @@
                     { "name": "condition", "type": "string", "optional": true, "description": "Expression to use as a breakpoint condition. When specified, debugger will only stop on the breakpoint if this expression evaluates to true." }
                 ],
                 "returns": [
-                    { "name": "breakpointId", "$ref": "BreakpointId", "description": "Id of the created breakpoint for further reference." },
+                    { "name": "breakpointId", "$ref": "BreakpointId", "optional": true, "description": "Id of the created breakpoint for further reference." },
```

Comment:
Ideally we shouldn't be changing the protocol definition here.  It doesn't look like Chrome treats this as optional, so I don't think we should either.

Reply:
I agree that it's not documented in the protocol.  But if you look at the VS Code source I referenced, you'll see that this is the way that Node (at least) actually appears to work - i.e., this is Node's actual protocol, even if it's not the documented protocol.  So, VS Code + Node bug?  Protocol spec bug?